pkg/finality-grandpa/dummy_chain_test.go

```diff
@@ -20,7 +20,6 @@ type blockRecord struct {
 	parent string
 }
 
-// dummyChain is translation of finality_grandpa::testing::chain::DummyChain
```

Comment:
can add a helpful comment here

Reply:
I think it's understood that this implements the `Blockchain` interface.